File: my-folder/0412-fizz-buzz/solution.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let tempArray = []

    for(let i=1; i<=n; i++){
        if(i%3===0 && i%5===0){
            tempArray.push("FizzBuzz")
        } else if(i%3===0){
            tempArray.push("Fizz")
        } else if(i%5===0){
            tempArray.push("Buzz")
        } else{
            tempArray.push(String(i));
        }
    }
    return tempArray
    
};

File: my-folder/0412-fizz-buzz/solution.rs
impl Solution {
    pub fn fizz_buzz(n: i32) -> Vec<String> {
        let mut tempArr = Vec::new();

    for i in 1..=n {
        if i % 15 == 0 {
            tempArr.push("FizzBuzz".to_string());
        } else if i % 3 == 0 {
            tempArr.push("Fizz".to_string());
        } else if i % 5 == 0 {
            tempArr.push("Buzz".to_string());
        } else {
            tempArr.push(i.to_string());
        }
    }
    tempArr
}
    
}

